packages/graphql/src/schema/get-alias-meta.test.ts

```diff
@@ -0,0 +1,127 @@
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FieldDefinitionNode } from "graphql";
+import getAliasMeta from "./get-alias-meta";
+
+describe("getAliasMeta", () => {
+    test("should return undefined if no directive found", () => {
+        // @ts-ignore
+        const field: FieldDefinitionNode = {
+            directives: [
+                {
+                    // @ts-ignore
+                    name: { value: "RANDOM 1" },
+                },
+                {
+                    // @ts-ignore
+                    name: { value: "RANDOM 2" },
+                },
+                {
+                    // @ts-ignore
+                    name: { value: "RANDOM 3" },
+                },
+                {
+                    // @ts-ignore
+                    name: { value: "RANDOM 4" },
+                },
+            ],
+        };
+
+        const result = getAliasMeta(field);
+
+        expect(result).toBeUndefined();
+    });
+
+    test("should throw 'property required'", () => {
+        const field: FieldDefinitionNode = {
+            directives: [
+                {
+                    // @ts-ignore
+                    name: { value: "alias", arguments: [] },
+                },
+                {
+                    // @ts-ignore
+                    name: { value: "something else" },
+                },
+            ],
+        };
+
+        expect(() => getAliasMeta(field)).toThrow("@alias property argument is required");
+    });
+
+    test("should throw property argument is not a string", () => {
+        const field: FieldDefinitionNode = {
+            directives: [
+                {
+                    // @ts-ignore
+                    name: {
+                        value: "alias",
+                        // @ts-ignore
+                    },
+                    arguments: [
+                        {
+                            // @ts-ignore
+                            name: { value: "property" },
+                            // @ts-ignore
+                            value: { kind: "NOT A STRING!" },
+                        },
+                    ],
+                },
+                {
+                    // @ts-ignore
+                    name: { value: "RANDOM 2" },
+                },
+            ],
+        };
+
+        expect(() => getAliasMeta(field)).toThrow("@alias property argument was not a string");
+    });
```

Comment:
These tests really belong in `packages/graphql/src/schema/validation/validate-document.test.ts`, where you will be able to test using actual type definitions, like the other tests in there.